code-execution/src/main/java/com/dojo/codeexecution/service/GitManager.java

```diff
@@ -81,4 +86,27 @@ private String getRepositoryNameByOwner(String username, String game) throws IOE
     private GHUser getGitHubUser(String username) throws IOException {
         return gitHub.getUser(username);
     }
+
+    public String deleteReposForParticularGame(String game) throws IOException {
+        GHMyself account = gitHub.getMyself();
+        Collection<GHRepository> listOfAllRepos = account.getAllRepositories().values();
```

Comment:
listOfAllRepos can be just reposm, imagine if we did everywhere setof..., mapof.., etc.